src/components/Cewl/Cewl.tsx

```diff
@@ -162,121 +234,138 @@ const Cewl = () => {
-                                        label={"Password"}
-                                        required
-                                        style={{ flex: 1 }}
-                                        {...form.getInputProps("password")}
-                                    />
-                                </div>
-                            </>
-                        )}
-                        {isDigestAuth && (
-                            <>
-                                <TextInput label={"Username"} required {...form.getInputProps("username")} />
-                                <TextInput label={"Password"} required {...form.getInputProps("password")} />
-                            </>
-                        )}
-                        <Checkbox
-                            label={"Include words with numbers"}
-                            checked={checkedNumber}
-                            onChange={(e) => setCheckedNumber(e.currentTarget.checked)}
-                        />
-                        <Checkbox
-                            label={"Show only lowercase words"}
-                            checked={checkedLowercase}
-                            onChange={(e) => setCheckedLowercase(e.currentTarget.checked)}
-                        />
-                        <Checkbox
-                            label={"Show the count for earch word found"}
-                            checked={checkedCount}
-                            onChange={(e) => setCheckedCount(e.currentTarget.checked)}
-                        />
-                        <Checkbox
-                            label={"Include and show email addresses"}
-                            checked={checkedEmail}
-                            onChange={(e) => setCheckedEmail(e.currentTarget.checked)}
-                        />
-                    </>
-                )}
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <Button type={"submit"}>Scan</Button>
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-            </Stack>
-        </form>
+                            <Checkbox
+                                label={"Show the count for earch word found"}
+                                checked={checkedCount}
+                                onChange={(e) => setCheckedCount(e.currentTarget.checked)}
+                            />
+                            <Checkbox
+                                label={"Include and show email addresses"}
+                                checked={checkedEmail}
+                                onChange={(e) => setCheckedEmail(e.currentTarget.checked)}
+                            />
+                        </>
+                    )}
+                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
+                    <Button type={"submit"}>Scan</Button>
```

Comment:
The format for baselining is Start <title of tool> 
```suggestion
                    <Button type={"submit"}>Start {title}</Button>
```